jobs/db_store.go

```diff
@@ -338,11 +348,32 @@ func (s *DatabaseStore) scanParams(method string, params []byte) (interface{}, e
 	}
 }
 
+func (s *DatabaseStore) scanResult(method string, result []byte) (interface{}, error) {
+	if result == nil {
+		return nil, nil
```

Comment:
I was thinking that then a `result.(ReleaseResult)` would always work, even if metadata were nil.